opt_PXL.ino

```diff
@@ -70,7 +71,8 @@ static const unsigned char PROGMEM logo16_glcd_bmp[] =
 Adafruit_NeoPixel strip = Adafruit_NeoPixel(STRIP_LENGTH, NPPin, NEO_GRB + NEO_KHZ800);
 
 // Variable assignments for the Keypad
-int adc_key_val[5] ={ 0, 331, 509, 145, 744 };  
+PROGMEM const unsigned int adc_key_val[5] = { 0, 331,509, 145,744 };

```

Comment:
Should this be in `PROGMEM`? Gut feel tells me it'll mess up the key values.

Reply:
It was working in this setup. I was just looking at trying to save as much memory space as possible and then slowly fleshing it back out. Definately could be put back to original. This may solve all my issues as I guess when they are in progmem they can't be changed or something funky